packages/sdks/snippets/vue/src/components/custom-components/custom-tabs/CustomTabs.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <div class="dynamics-slots" v-if="tabList.length">
+      <button
+        v-for="(tab, index) in tabList"
+        :key="index"
+        :class="{ 'active': activeTab === index }"
+        @click="activeTab = index"
+      >
+        {{ tab.tabName }}
+      </button>
+
+      <Blocks
+        :blocks="tabList[activeTab].blocks"
+        :path="`tabList.${activeTab}.blocks`"
+        :parent="builderBlock.id"
+      />
+    </div>
+  </template>
+  
+  <script setup lang="ts">
+  import { ref, defineProps } from 'vue';
+  import { Blocks, type BuilderBlock } from '@builder.io/sdk-vue';
+
+  const activeTab = ref<number>(0);
```

Comment:
type generic is unnecessary when the value can be inferred from the initial value, which is the case here:

```suggestion
  const activeTab = ref(0);
```